### PR TITLE
chore: Use documentation as built in the publication branch of the devfile/docs repository

### PR DESCRIPTION
**Pull request by themr0c**
Use documentation as built in the publication branch of the devfile/docs repository

Signed-off-by: Fabrice Flore-Thébault <ffloreth@redhat.com>

### CODE REVIEW

**Comment by themr0c** [comment]
It seems GitHub doesn't allow changing container during a workflow.

It would be safer to build the docs in the docs repository, and only copy the build output in this repository.

**Comment by schultzp2020** [comment]
@themr0c Ill look at this today.

File: .github/workflows/BuildAndDeploy.yaml [hunk 1]
```diff
@@ -34,14 +30,10 @@ jobs:
...
-      - name: Install docs deps
-        run: yarn --cwd ./docs install
-      - name: Generate docs api references
-        run: yarn --cwd ./docs generate-api-reference
       - name: Build docs site with antora
-        uses: kameshsampath/antora-site-action@master
-        with:
-          antora_playbook: docs/antora-playbook.yml
+        run: |
+          yarn add @antora/lunr-extension
```
Comment (schultzp2020) [inline .github/workflows/BuildAndDeploy.yaml:35]: I don't think this is necessary. If you run this command locally in `/docs`, the package will automatically be installed when `yarn install` is executed.

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

**Review by schultzp2020 — DISMISSED** [review]

**Comment by themr0c** [comment]
Thanks for the approval @schultzp2020. 

The merging is still blocked, we need actions from yourself or a maintainer:

* Clear the "1 change requested".
* Approve running workflows to clear: "First-time contributors need a maintainer to approve running workflows."

**Comment by themr0c** [comment]
@schultzp2020 can you merge the pull request? (I can't)

**Review by schultzp2020 — APPROVED** [review]

**Pull request merged**